plugins/modules/system/sysupgrade.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2020, Andrew Klaus <andrewklaus@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sysupgrade
+
+short_description: Manage OpenBSD system upgrades
+
+version_added: "2.10"
+
+description:
+    - This module can be used to manage OpenBSD system upgrades using sysupgrade.
+
+options:
+    snapshot:
+        description:
+            - Apply the latest snapshot.
+            - Otherwise release will be applied.
+        default: false
```

Comment:
```suggestion
        default: no
```
and so on
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#examples-block
`If your examples use boolean options, use yes/no values. Since the documentation generates boolean values as yes/no, having the examples use these values as well makes the module documentation more consistent.`